src/lib/support/verhoeff/Verhoeff10.cpp

```diff
@@ -30,6 +30,7 @@
 
 #ifndef VERHOEFF10_NO_MULTIPLY_TABLE
 
+// clang-format off
```

Comment:
What motivates turning off formatting here?

Reply:
Clang-format will attempt to reformat the `sMultiplyTable` below to fill the allotted columns (I believe the fill line is set at around 132 columns).  This loses the readability of the table that is laid out to match Verhoeff encoding base.